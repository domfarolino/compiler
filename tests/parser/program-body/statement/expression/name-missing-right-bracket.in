program dominic is

  integer dominic [0 : 9];
  integer farolino;

begin

  farolino := dominic[3 + 1 + 4;

end program.
