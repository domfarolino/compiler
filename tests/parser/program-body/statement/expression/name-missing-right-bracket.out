[1;31mLine 8: Expected ']' after expression in name, but got: ';'
Line 8: Error parsing program body
[0m